### PR TITLE
WDIO Tests

### PR DESCRIPTION
**Pull request by tstetsyk**
Hi Artem! Please review my PR.
Thanks!

### CODE REVIEW

**Comment by 4rtemOv** [comment]
Привіт, мій інвайт чи то заекспайрився чи ти його відмінила( можешь плз спробувати ще раз?

File: test/specs/test.e2e.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { expect, browser, $ } from '@wdio/globals'
+
+describe('Inventory Page', () => {
+    it('Verify elements are displayed on the page', async () => {
+        await browser.url(`https://www.saucedemo.com/`)
```
Comment (4rtemOv) [inline test/specs/test.e2e.js]: Great job!
Let's make your code better.
You can use beforeAll function - more about it here https://stackoverflow.com/questions/46799242/before-hook-before-all-test-cases-in-mocha

Comment (4rtemOv) [inline test/specs/test.e2e.js]: `https://www.saucedemo.com/` can be added to the config as baseUrl - find and read about it here https://webdriver.io/uk/docs/configurationfile/

**Review by 4rtemOv — CHANGES_REQUESTED** [review]
Several small comments were left

**Review by ovartem — APPROVED** [review]
well done

**Pull request merged**